feat: Git provider integration at organization level PLUTO-535 (#1706)
codacy/docs: docs/assets/includes/default-git-provider-settings-tip.md

!!! tip

>     Configure the [default settings](../../organizations/default-git-provider-integration.md) for the integration with your Git provider to ensure that Codacy uses the same settings when enabling the Git provider integration for newly added repositories.

Reviewer: I feel like this sentence is a bit hard to understand.

What do you think of this alternative?

```suggestion
    Configure the [default Git provider integration settings](../../organizations/default-git-provider-integration.md) that Codacy applies to new repositories to help ensure that all new repositories have the same settings.
```

Author: Yep, much simpler! 🤓 👍

>     You can configure the [default settings](../../organizations/default-git-provider-integration.md) for the integration with your Git provider to ensure that Codacy will use the same settings when enabling the Git provider integration for newly added repositories.

Reviewer 2: A couple tweaks:

```suggestion
    Configure the [default settings](../../organizations/default-git-provider-integration.md) for the integration with your Git provider to ensure that Codacy uses these settings when enabling the Git provider integration for newly added repositories.
```
